services/tenant-ui/frontend/src/assets/style.scss

```diff
@@ -1,4 +1,5 @@
 @import '_theme.scss'; // copied from '~primevue/resources/themes/nova/theme.css'
 @import 'layout.scss';
 @import 'forms.scss';
-@import 'primevueComponents.scss'
\ No newline at end of file
+@import 'primevueComponents.scss';
+@import 'statusChip.scss';
```

Comment:
Could import this one in the one above so the top level styles file doesn't end up with all the specific primevue imports if we go that way

Reply:
just let me know how you want to re-shuffle this stuff.